src/gui/qgsattributeeditorcontext.h

```diff
@@ -132,6 +133,20 @@ class GUI_EXPORT QgsAttributeEditorContext
      */
     inline QgsMapCanvas *mapCanvas() const { return mMapCanvas; }
 
+    /**
+     * Sets the associated CAD dock widget, \a cadDockWidget, (e.g. to be used in map tools).
+     * \see cadDockWidget()
+     * \since QGIS 3.10
+     */
+    void setCadDockWidget( QgsAdvancedDigitizingDockWidget *cadDockWidget ) { mCadDockWidget = cadDockWidget; }
+
+    /**
+     * Returns the associated map canvas (e.g. to zoom to related features).
+     * \see setCadDockWidget()
+     * \since QGIS 3.10
+     */
+    inline QgsAdvancedDigitizingDockWidget *cadDockWidget() const { return mCadDockWidget; }
```

Comment:
What's the reason for inlining this?

Reply:
It was to be consistent with other methods declaration (except I forgot to put it in front on setCadDockWidget), but I'm pretty sure this is useless here. 

I removed it.